scalableminds/webknossos
# backend conversion route for tracings to hybrid

- create a make-hybrid route which lets you turn any non-hybrid tracing into a hybrid tracing 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create a non-hybrid explorational tracing and then send a `PATCH` request to `http://localhost:9000/api/annotations/Explorational/:id/makeHybrid`
- reload tracing
- tracing should now be a hybrid tracing

### Issues:
- begins with #3288

------
~~- [ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
~~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
~~- [ ] Updated [documentation](../blob/master/docs) if applicable~~
~~- [ ] Needs datastore update after deployment~~
